src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -128,8 +128,64 @@ public AD7DebugSession(Stream debugAdapterStdIn, Stream debugAdapterStdOut, List
             m_dataBreakpoints = new Dictionary<string, IDebugPendingBreakpoint2>();
             m_exceptionBreakpoints = new List<string>();
             m_variableManager = new VariableManager();
+
+            //Register sendInvalidate request
+            Protocol.RegisterRequestType<SendInvalidateRequest, SendInvalidateArguments, SendInvalidateResponse>(r => this.HandleSendInvalidateRequestAsync(r));
+    
         }
 
+        private void HandleSendInvalidateRequestAsync(IRequestResponder<SendInvalidateArguments> responder)
+        {
+            InvalidatedEvent invalidated = new InvalidatedEvent();
+            SendInvalidateResponse response = new SendInvalidateResponse();
+            
+            // Setting the area and adding it to the result
+            response.body.result += "sendInvalidate: { Area: ";
```

Comment:
I'm not a fan of creating json strings by hand, can you define this as a class that can be serialized by the JSON library?

Reply:
I changed the output format a bit, now it's a serializable class, so there shouldn't be any problems